lib/rspec/core/configuration_options.rb

```diff
@@ -13,6 +13,10 @@ def initialize(args)
         organize_options
       end
 
+      def command_line_and_env_args
```

Comment:
It's because I was transferring all the env args to the CLI args of the child processes and removing SPEC_OPTS completely for the children, but there's actually no need to do that, so this can be removed.